server/builtin/debug.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package builtin
+
+import (
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/config"
+	"github.com/mattermost/mattermost-plugin-apps/server/proxy"
+	"github.com/mattermost/mattermost-plugin-apps/utils"
+)
+
+func (a *builtinApp) debugCommandBinding() apps.Binding {
+	return apps.Binding{
+		Label:    "debug",
+		Location: "debug",
+		Bindings: []apps.Binding{
+			apps.Binding{
+				Label:       "clean",
+				Location:    "clean",
+				Hint:        "",
+				Description: "remove all Apps and reset the persistent store",
+				Call: &apps.Call{
+					Path: pDebugClean,
+				},
+				Form: &noParameters,
+			},
+			apps.Binding{
+				Label:       "bindings",
+				Location:    "bindings",
+				Description: "Display all bindings for the current context",
+				Call: &apps.Call{
+					Path: pDebugBindings,
+				},
+				Form: &noParameters,
+			},
+		},
+	}
+}
+
+func (a *builtinApp) debugClean(creq apps.CallRequest) apps.CallResponse {
+	_ = a.conf.MattermostAPI().KV.DeleteAll()
```

Comment:
Maybe I am wrong, but seems like the only "sysadmin" check is when providing the bindings. That means that I could just curl the endpoint with any user token and clean the database. Am I right? Should we add some more checks here? (or in a middleware in the handling level)

Reply:
Similar concerns with other endpoints.

Reply:
Argh, I think you're right. I will fix it, of course, but really... it bothers me. I wanted a shortcut - using direct `go` interfaces from `builtin` but I question it now. Perhaps, the simplest/best course of action is to have `builtin` be a true app in the sense of needing to expand the admin token, and then using `appsclient` to invoke the necessary APIs.

Reply:
Done, somewhat idiomatically for apps, with the help of Expand.